addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -233,6 +319,56 @@ weSnippetEditor.SnippetsMenu.include({
             throw e;
         }
     },
+    /**
+     * @private
+     */
+    _onAnimateTextClick(ev) {
+        const sel = window.getSelection();
+        const editable = this.options.wysiwyg.$editable[0];
+        if (!sel.rangeCount) {
+            return;
+        }
+        const range = getDeepRange(editable, { splitText: true, select: true, correctTripleClick: true });
+        if (this.$currentAnimatedText.length) {
+            this.$currentAnimatedText.contents().unwrap();
+            this.options.wysiwyg.odooEditor.historyResetLatestComputedSelection();
+            this._toggleHighlightAnimatedTextButton();
+            ev.target.classList.remove('active');
+            this.options.wysiwyg.odooEditor.historyStep();
+        } else {
+            if (sel.getRangeAt(0).collapsed) {
+                return;
+            }
+            const animatedTextEl = document.createElement('span');
+            animatedTextEl.classList = 'o_animated_text o_animate o_animate_preview o_anim_fade_in';
```

Comment:
Not sure what's this black magic if it works, but the doc says classList is supposed to be a read-only property (did not even know it could be read `The Element.classList is a read-only property that returns a live DOMTokenList collection of the class attributes of the element.` https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Use `className` or `classList.add(class1, class2, class3)`

Reply:
FYI if you grep ".classList ="  you will see that Kamen has done the same thing several times in the commit "filter products by price"

Reply:
@kzhekov Here is a bug report :point_up: for the commit: https://github.com/odoo/odoo/commit/41f7c7bc6428050d4c7956edca26b7c1bd2d8e04, if you can make a PR to avoid that bad practice which may not work on all browsers, thanks :+1: You can ping me on it when it is done :+1:

Reply:
@bvr-odoo @qsm-odoo Thanks for the info, didn't know it was supposed to be read-only 👍🏻 Will fix it